### DISCUSSION TITLE
Click cancel in Sign in chrome prompt

### DISCUSSION THREAD

**Discussion by sushixdd** [cypress-io/cypress#15638] [Questions and Help]
I'm a newbie in Cypress and so far I'm having a fun with it and I have no problem finding answers on my own, but I spent a considerable time googling this, couldn't find a solution
![cancel_prompt](https://user-images.githubusercontent.com/64040563/112256421-08f7ae00-8c64-11eb-8077-d6cec61b78ce.png)

Basically, when the site is asking me to submit a sign form, I need to click cancel, but I haven't found a way to make it work
I know about singing in using a username/password combination, like https://username:pw@mypage123.com, but that basically performs a 'Sign In' action, I need to just click 'Cancel'.

edit: I'm using Chrome 89 and my Cypress version should be 6.8.0

**Comment by herecydev** [comment]
What's prompting that sign in request to appear? Given that chrome is rendering that outside of the DOM I don't think you'll be able to access that at all.

**Reply by sushixdd** [reply]
Thanks for the answer
visiting the site does. It returns 401 on failed login (obviously), but on the cancel action on the sign in prompt (when manually testing), the prompt is gone and site is just blank with a string "Unauthorized access". My goal was to achieve this result (since the 401 is an expected outcome), compare the string using cy.contains() or something and be done with it.
Anyways, this is what I was afraid of, that there's no way to interract with the prompt, since it's not the DOM.
